admin/src/react-components/accounts.js

```diff
@@ -10,32 +12,62 @@ import {
   TextField,
   DateField,
   BooleanField,
+  ReferenceManyField,
   EditButton,
   Filter
 } from "react-admin";
 
+const styles = {
+  hide: { display: "none" },
+  noBorder: { border: "0px" }
+};
+
 const AccountFilter = props => (
   <Filter {...props}>
     <TextInput label="Search ID" source="_text_id" alwaysOn />
   </Filter>
 );
 
-export const AccountList = props => (
-  <List {...props} filters={<AccountFilter />}>
-    <Datagrid>
-      <TextField source="id" />
-      <DateField source="inserted_at" />
-      <DateField source="updated_at" />
-      <BooleanField source="is_admin" />
-      <EditButton />
-    </Datagrid>
-  </List>
-);
+export const AccountList = withStyles(styles)(props => {
+  const { classes } = props;
 
-export const AccountEdit = props => (
-  <Edit {...props}>
-    <SimpleForm>
-      <BooleanInput source="is_admin" />
-    </SimpleForm>
-  </Edit>
-);
+  return (
+    <List {...props} filters={<AccountFilter />}>
+      <Datagrid>
+        <TextField source="id" />
+        <DateField source="inserted_at" />
+        <DateField source="updated_at" />
+        <ReferenceManyField label="Identity" target="_account_id" reference="identities">
```

Comment:
Why not just a `ReferenceField`?

Reply:
So this is actually a little confusing in the naming, ReferenceField lets you fetch a record via a foreign key on the current record, whereas ReferenceManyField lets you fetch N records in the other direction (where a foreign key in another table references this record.) (There's no equivalent of "has_one", so you just list one record in this case)